src/algokit_utils/transactions/transaction_composer.py

```diff
@@ -461,42 +477,66 @@ def send(self, params: SendParams | None = None) -> SendTransactionComposerResul
         params = params or SendParams()
 
         # Update config from params if provided
+        cover_flag = params.get("cover_app_call_inner_transaction_fees")
+        populate_flag = params.get("populate_app_call_resources")
+        effective_cover = bool(cover_flag) if cover_flag is not None else False
```

Comment:
This can be simplified as `bool(None)` will result in `False`
```suggestion
        effective_cover = bool(cover_flag)
```